### DISCUSSION TITLE
Query loop vs CPT (Custom Post Types)

### DISCUSSION THREAD

**Discussion by crs1138** [WordPress/gutenberg#42422] [Q&A] [answered]
Is there currently any way to use a custom post type query in the block themes? Or do I still have to have a hybrid theme and query CPTs using PHP?

**Comment by tnchuntic** [comment, marked as answer]
Query loop will be able to display the CPT available on your website. 

It is found under 
> Query loop block > Settings 
> Switch "Inherit query from template" off  
> Choose the CPT you want to display.

Hope it helps.
